README.md

```diff
@@ -289,19 +289,28 @@ By default, both `pip-compile` and `pip-sync` will look first
 for a `.pip-tools.toml` file and then in your `pyproject.toml`. You can
 also specify an alternate TOML configuration file with the `--config` option.
 
-For example, to by default generate `pip` hashes in the resulting
-requirements file output, you can specify in a configuration file
+For example, to by default annotate dependencies with their origins in the resulting
+requirements file output, you can specify in a configuration file:
 
 ```toml
 [tool.pip-tools]
-generate-hashes = true
+annotate = true
```

Comment:
The `annotate = true` setting is actually the default value which typically doesn't have any extra effect and is rarely used in practice in this way. On the other hand, the 'generate hashes' option is quite good example. Perhaps we could keep it in our documentation?